clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -264,6 +264,71 @@ def ConstPtrAttr : CIR_Attr<"ConstPtr", "ptr", [TypedAttrInterface]> {
   let hasCustomAssemblyFormat = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// ThreeWayCmpInfoAttr
+//===----------------------------------------------------------------------===//
+
+def ThreeWayCmpOrderingKind_Strong : I32EnumAttrCase<"Strong", 1, "strong">;
+def ThreeWayCmpOrderingKind_Partial : I32EnumAttrCase<"Partial", 2, "partial">;
```

Comment:
Can we make these bit less verbose? `ThreeWayCmpOrderingKind_Strong` -> `CmpStrong`, etc...

Reply:
Updated.